tasks/debian.yml

```diff
@@ -1,22 +1,40 @@
 ---
 - name: debian | installing pre-reqs
-  apt:
+  package:
     name:
       - lvm2
       - scsitools
-      - xfsprogs
     state: "present"
   become: true
 
+- name: centos | install xfs tools
+  package:
+    name: "xfsprogs"
+    state: "present"
+  become: yes
+  loop: "{{ lvm_groups|subelements('lvnames') }}"
```

Comment:
It's a shame that this could run multiple times. It's not that big a deal though.

Reply:
Sure, do you know how to assess that there's a match in a list?

Reply:
Two options I can think of:
1. add a task that uses this loop with set_fact, then install the package based on the fact.
2. some complicated unreadable jinja expression using a few `selectattr`s.

The former is probably preferable :)

Reply:
Not a fan of set fact: to me you should use it to add a machine attribute (os kind or hardware), not something related to installed software.
May be with a register somehow